components/edit/configure.tsx

```diff
@@ -1,35 +1,34 @@
-import { useCallback, useContext } from 'react';
+import { gatewayTool } from '@/actions/knowledge/util';
```

Comment:
Seems strange that these imports changed with one very minor code change.

Reply:
Part of this has to do with resolved merge conflicts, and some of the changes are from me fixing Prettier on my machine